apps/functions/githubWebhook/index.ts

```diff
@@ -0,0 +1,29 @@
+import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin';
+import {handlePullRequestEvent} from './pull-request';
+
+admin.initializeApp({...functions.firebaseConfig()});
+
+export const githubWebhook = functions
```

Comment:
Nit: I think every function index should have a small little doc outlining the goal